composer.json

```
@@ -9,8 +9,9 @@
     },
     "require-dev": {
         "doctrine/coding-standard": "^9",
-        "phpunit/phpunit": "^7.5|^8.5|^9.5",
         "phpstan/phpstan": "1.4.10 || 1.10.15",
+        "phpstan/phpstan-phpunit": "^1.3",
```

This might fix it, but no guarantee. Also, should we pin these dependency as well?

```suggestion
        "phpstan/phpstan-phpunit": "^0.12.22 || ^1.3",
```